pydough/sqlglot/transform_bindings/mysql_transform_bindings.py

```diff
@@ -694,3 +696,56 @@ def convert_ordering(
                 expression=sqlglot_expressions.Identifier(this="utf8mb4_bin"),
             )
         return arg
+
+    def create_empty_singleton(self) -> SQLGlotExpression:
+        return (
+            sqlglot_expressions.Select()
+            .select(sqlglot_expressions.Star())
+            .from_(
+                sqlglot_expressions.values(
+                    [
+                        sqlglot_expressions.Anonymous(
+                            this="ROW",
+                            expressions=[sqlglot_expressions.convert((None,))],
+                        )
+                    ]
+                )
+            )
+        )
+
+    def convert_user_generated_range(
+        self,
+        collection: RangeGeneratedCollection,
+    ) -> SQLGlotExpression:
+        column_name: SQLGlotExpression = sqlglot_expressions.Identifier(
+            this=collection.column_name, quoted=False
+        )
+
+        table_name: SQLGlotExpression = sqlglot_expressions.Identifier(
+            this=collection.name, quoted=False
+        )
+
+        # Empty range if step == 0 or (step > 0 and start >= end) or
+        # (step < 0 and start <= end)
+        empty_range = (
+            (collection.step > 0 and collection.start >= collection.end)
+            or (collection.step < 0 and collection.start <= collection.end)
+            or (collection.step == 0)
+        )
```

Comment:
This is duplicated code from base bindings. 
Let's have other part in a separate function and override that function instead of duplicating the code for `convert_user_generated_range`